packages/templating/plugin/html_scanner.js

```diff
@@ -169,8 +171,15 @@ html_scanner = {
           templateDotNameLiteral + ", " + renderFuncCode + ");\n";
       } else {
         // <body>
-        if (hasAttribs)
-          throwParseError("Attributes on <body> not supported");
+        if (hasAttribs) {
+          for(var attr in attribs) {
```

Comment:
Thought about it but the plugin doesn't currently use underscore as declared in the package.js and I wasn't sure if it's needed if I only need it here. What do you think?